src/styles/defaultTheme.js

```diff
@@ -22,8 +33,35 @@ export default {
             main: COLOR_WARNING,
         },
         background: {
-            primary: COLOR_WHITE,
+            main: COLOR_WHITE,
             secondary: '#f9fafc',
+            disabled: COLOR_GRAY_1,
+        },
+        text: {
+            main: COLOR_DARK_1,
+            title: COLOR_GRAY_4,
+            header: COLOR_GRAY_3,
+            label: COLOR_GRAY_4,
+            disabled: COLOR_GRAY_2,
+        },
+        border: {
+            main: COLOR_GRAY_3,
+            divider: COLOR_GRAY_2,
+            disabled: COLOR_GRAY_2,
+        },
+        action: {
+            hover: COLOR_GRAY_1,
+        },
+        shadows: {
+            brand: `0 0 2px ${COLOR_BRAND}`,
+            success: `0 0 2px ${COLOR_SUCCESS}`,
+            error: `0 0 2px ${COLOR_ERROR}`,
+            shadow_1: `0 0 2px 0 ${COLOR_GRAY_3}`,
+            shadow_2: `0 2px 4px 0 ${COLOR_GRAY_2}`,
+            shadow_3: `0 0 1px 0 ${COLOR_GRAY_3}`,
+            shadow_4: `0 1px 2px 0 ${COLOR_GRAY_2}`,
+            shadow_5: `0 0 3px ${COLOR_GRAY_1}`,
+            shadow_6: `0 2px 12px 0 ${COLOR_GRAY_2}`,
         },
```

Comment:
Good job, but I think we should put this outside palette since shadows are not simply colors